### PR TITLE
Turkish translations updated

### PR DESCRIPTION
**Pull request by Emefar**
Added new variables (for v4.0.6) and changed 1 line.

### CODE REVIEW

File: source/core/locale/tr.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * Name : Turkish (Türkiye)
  * Code : TR
  * Tag  : tr-TR
- * Translator: Xus (s25), Anonim, Falconrock
+ * Translator: Xus (s25), Anonim, Falconrock, Emefar (s33)
```
Comment (Emefar) [inline source/core/locale/tr.js]: My old nickname is Falconrock. If you want delete Falconrock.

**Comment by GramThanos** [comment]
Great work @Emefar !

**Comment by Emefar** [comment]
Thank you! @GramThanos 👍

**Comment by GramThanos** [comment]
About your old nickname "Falconrock", it is your call.
We can remove it or leave it, what ever you want.

**Comment by Emefar** [comment]
I am not using Falconrock nickname. I am using Emefar now. Please delete Falconrock.

**Pull request merged**